package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
 	"name": "elementor",
-	"version": "3.21.0",
+	"version": "3.22.0",
```

Comment:
Please don't make any changes to this file.

Reply:
@hein-obox I have not made any changes, this was apparently changed when building scripts and styles. Should I roll back this change?

Reply:
done